### ISSUE TITLE
Odd cylinders in Ironforge

### ISSUE THREAD

**Issue by timkurvers** [timkurvers/wowser#105]
![screen shot 2016-01-18 at 19 37 56](https://cloud.githubusercontent.com/assets/378235/12404659/b0917718-be3d-11e5-8f9e-dc1ffbbfb24b.png)

**Comment by fallenoak** [comment]
I'm pretty sure those are using transparency and uv animations to replicate steam. I had transparencies kind of working a while ago, but junked it when I refactored M2s.

See here for what I mean by steam:
http://vignette3.wikia.nocookie.net/wowwiki/images/d/d3/The_Great_Forge.jpg/revision/latest?cb=20100123225748

**Comment by timkurvers** [comment]
Ah, great reference, that has to be it :+1: Tracked by #97 and #98 in that case.

**Comment by fallenoak** [comment]
I'd keep this one open, just in case there's more than that at play.

**Comment by fallenoak** [comment]
Looks like this also involves particle emitters!

**Comment by timkurvers** [comment]
Is it the steam?

**Comment by fallenoak** [comment]
Yeah, it's definitely the steam. The model is ```World\\KHAZMODAN\\IRONFORGE\\PASSIVEDOODADS\\LavaSteam\\LavaSteam.m2```. It turns out the transparency is 0.0 for the entire animation block, and it has several particle emitters.

My theory is that the cylinders are geometry meant to aid in the positioning of the particles, and the steam from my screenshot reference above is actually particle emitters.
